src/mlpack/methods/kmeans/kmeans_plus_plus_initialization.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file kmeans_plus_plus_initialization.hpp
```

Comment:
```suggestion
 * @file methods/kmeans/kmeans_plus_plus_initialization.hpp
```

Reply:
Ahh, nice, thank you.  I need to update the header guard too.